packages/input/src/FzCurrencyInput.vue

```diff
@@ -18,4 +18,9 @@ onMounted(() => {
   inputRef.value = fzInputRef.value.inputRef;
 });
 const model = defineModel("amount");
+
+defineExpose({
+  inputRef: fzInputRef.value?.inputRef,
```

Comment:
are you sure this will be the inputRef and not null/undefined? I think defineExpose is executed before fzInputRef is populated so the value will be null

Reply:
yes I'm looking at it right now, indeed doesn't work

Reply:
why not exposing the ref instead of the value?